index.js

```diff
@@ -922,6 +922,12 @@ export function load_placements(force_load = false) {
   return Promise.all(
     elements.map((element, index) => {
       const placement = Placement.from_element(element);
+
+      if (!placement) {
+        // Placement has already been loaded
```

Comment:
Worth debug logging here?

Reply:
This will probably have already logged here: https://github.com/readthedocs/ethical-ad-client/blob/5d6db00555cfbdded49c8a58ce88647c64b5ade7/index.js#L487.